### ISSUE TITLE
[Feature] scroll to top should be in side wise.

### ISSUE THREAD

**Issue by shyamtawli** [FreshRSS/freshrss.org#59]
**Is your feature request related to a problem? Please describe.**
Scroll to top will look more attractive when on right or left side.

**Describe the solution you’d like**
Move to left.

**Describe alternatives you’ve considered**
A clear and concise description of any alternative solutions or features you’ve considered.

**Additional context**
i can work on it but help me finding the frontend files.

**Comment by Alkarex** [comment]
Maybe something for @marienfressinaud

**Comment by marienfressinaud** [comment]
Mmm, it seems to me it's a matter of personal preference. I don't understand the problem (if any) of having this button in the middle of the screen :thinking:

**Comment by shyamtawli** [comment]
@marienfressinaud yeah I know it depends on people to people but majority of site have it sidewise. So it may look good.

**Comment by marienfressinaud** [comment]
Well, I kind of like it in the middle of the screen, I think it balances the page nicely. But if you really want to move it, go ahead. You'll find the declarations to place the button here: https://github.com/FreshRSS/freshrss.org/blob/main/static/style.css#L410-L411
